lib/core/bloc/bloc.dart

```diff
@@ -0,0 +1,4 @@
+abstract class Bloc {
```

Comment:
This is a IBloc right?

Reply:
yes Fran! i renamed the class to "IBloc" to prevent confusions. Thanks!